run_pacta.R

```diff
@@ -0,0 +1,382 @@
+# load packages----
+# library(dotenv)
+library(dplyr)
+library(pacta.multi.loanbook.analysis)
+library(r2dii.analysis)
+library(r2dii.data)
+library(r2dii.plot)
+
+
+# source helpers----
+source("expected_columns.R")
+source("plots.R")
+
+# load config----
+config <- config::get()
+
+input_path_scenario <- config$directories$dir_scenario
+input_dir_abcd <- config$directories$dir_abcd
+input_path_matched <- config$directories$dir_matched
+
+input_path_scenario_tms <- file.path(input_path_scenario, config$file_names$filename_scenario_tms)
+input_path_scenario_sda <- file.path(input_path_scenario, config$file_names$filename_scenario_sda)
+
+input_path_abcd <- file.path(input_dir_abcd, "abcd_final.csv")
+
+output_path <- config$directories$dir_output
```

Comment:
nit: see above, this would be `dir_output` now

Reply:
👍🏻 
- see #54